lib/simple_kafka_consumer/consumer.rb

```diff
@@ -11,15 +12,22 @@ def initialize(kafka_servers, zookeeper_servers, options = {})
         topic_name,
         options
       )
+      Signal.trap("INT") do
```

Comment:
if we're trapping on each instance, why are we setting it on the class, rather than as an instance variable?